packages/stable/indent-integration-okta/src/okta-group.ts

```diff
@@ -66,9 +72,9 @@ export class OktaGroupIntegration
     const method =
       event === 'access/grant'
         ? // If it's a grant event, add the user
-          'PUT'
+        'PUT'
```

Comment:
do we have a linter config I can use? my editor had some differing opinions in a lot of places and I'd like to prevent that.